src/main/java/com/example/demo/matcher/models/NewOrderParams.java

```diff
@@ -1,13 +1,23 @@
 package com.example.demo.matcher.models;
 
+import com.example.demo.validation.UUID;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.ToString;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+
 @ToString @AllArgsConstructor @Getter
 public class NewOrderParams {
+    @UUID(message = "Bad UUID format")
```

Comment:
Nice! Where are we anticipating to see this message?

Reply:
When there is a bad UUID in a new order, the API will return a BadRequest response and the body will include this error message

Reply:
Like so:
![image](https://user-images.githubusercontent.com/118171430/211869269-5525b768-78f0-4294-b058-10c453eb7ee0.png)